lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Table.js

```diff
@@ -205,6 +205,7 @@ qx.Class.define('callbackery.ui.plugin.Table', {
             this._form.addListener('changeData',function(e){
                 model.setFormData(e.getData());
                 if (this._loading == 0){ // only reload when data has been changed by human
+                    this._loadDataReadOnly();
```

Comment:
hmmm where the table is a descendant of the form class, and thus has a _loadDataReadOnly method, I would still want to call the _loadDataReadOnly method of the associated form, as it is the form I want to reload and not the table.
```suggestion
                    this._form._loadDataReadOnly();
```

Reply:
also a few lines below, there is another reloadData, one issued in response to clicking a reload button. I guess there a this._form__loadDataReadOnly would also be warrented.

Reply:
`this._form` is an instance of `callbackery.ui.form.Auto` which does not have the `_loadDataReadOnly` method. 

`_loadDataReadOnly` is only defined in `callbackery.ui.plugin.Form` (the parent of `callbackery.ui.plugin.Table`). As far as I can tell `Table` extends `Form` and does not include a separate instance of `Form` (compose pattern).

Reply:
ah :see_no_evil: for the _form ...

the comment on the missing second invocation of _loadDataReadonly remains ...

Reply:
Just pushed :-)